### ISSUE TITLE
a question about the validation

### ISSUE THREAD

**Issue by best-wei** [megvii-model/HINet#23]
hello, thanks for your excellent work, but i have a question about the validation set, the validation set of MPRNet is Rain100H, but your validation sets is all the five test sets, is it fair?

**Comment by mayorx** [comment]
Hi, best-wei,

I may not understand what you mean, could you please describe it in more detail?
Based on MPRNet's paper,  it evaluates all the five test sets, and here is the code: https://github.com/swz30/MPRNet/blob/main/Deraining/test.py#L43 .

**Comment by best-wei** [comment]
thanks for your reply, I mean the validation during the training process, is the fair or approriate to use the test sets for validation? thanks very much~

**Comment by mayorx** [comment]
> thanks for your reply, I mean the validation during the training process, is the fair or approriate to use the test sets for validation? thanks very much~

Hi, best-wei,

We follow the same convention as the previous best method [MPRNet](https://github.com/swz30/MPRNet/blob/main/Deraining/train.py#L145).
